Dockerfile

```diff
@@ -1,12 +1,25 @@
-FROM golang:1.15.3-buster AS builder
-RUN apt-get update -y
-RUN apt-get install -y libgpgme-dev
-WORKDIR /go/src/tagger
+#
+# Builder
+#
+
+FROM registry.fedoraproject.org/fedora:latest AS builder
```

Comment:
I love Fedora but I would prefer to go with an UBI image (conservative), what do you reckon?